lib/css_parser/parser.rb

```diff
@@ -263,7 +263,7 @@ def parse_block_into_rule_sets!(block, options = {}) # :nodoc:
       current_media_query = ''
       current_declarations = ''
 
-      block.scan(/([\\]?[{}\s"]|(.[^\s"{}\\]*))/).each do |matches|
+      block.scan(/(([\\]{2,})|([\\]?[{}\s"])|(.[^\s"{}\\]*))/).each do |matches|
```

Comment:
This will look for specifically the case where there is 2 or more backslashes, and capture them all as a single token without the quotes at the end.